crates/bevy_ecs/src/lib.rs

```diff
@@ -689,6 +690,7 @@ mod tests {
         assert_eq!(get_added::<B>(&mut world), vec![e1]);
 
         world.clear_trackers();
+        world.exclusive_system_counter = world.increment_global_system_counter();
```

Comment:
I think it probably makes sense to make world.clear_trackers() just increment global_system_counter internally. That way people using world directly dont need to think about counters at all.